tests/test_weighted_pandas.py

```diff
@@ -926,3 +926,11 @@ def test_set_weights(mcmc_wdf):
     mcmc_wdf.set_weights(None, inplace=True)
     assert id(mcmc_wdf) == mcmc_id
     assert not mcmc_wdf.isweighted()
+
+
+def test_drop_weights(mcmc_wdf):
+    assert mcmc_wdf.isweighted()
+    noweights = mcmc_wdf.drop_weights()
+    assert not noweights.isweighted()
+    assert_array_equal(noweights.drop_weights(), mcmc_wdf)
```

Comment:
Why this line? Shouldn't that be false?

Similar to the testing of the labelled dataframes, wouldn't it be better to work with both `mcmc_df` and `mcmc_wdf`?

Reply:
Adjusted to now check with assert_frame_equal, and to check that assert_array_equal follows for noweights and mcmc_wdf